decidim-meetings/app/assets/javascripts/decidim/meetings/admin/registrations_form.js.es6

```diff
@@ -4,10 +4,12 @@ $(() => {
   if ($form.length > 0) {
     const $registrationsEnabled = $form.find("#meeting_registrations_enabled");
     const $availableSlots = $form.find("#meeting_available_slots");
+    const $reservedSlots = $form.find("#meeting_reserved_slots");
 
     const toggleDisabledFields = () => {
       const enabled = $registrationsEnabled.prop("checked");
       $availableSlots.attr("disabled", !enabled);
+      $reservedSlots.attr("disabled", !enabled);
```

Comment:
What's this `!enabled` part? What's `enabled`?

Reply:
`!enabled` is  equal to `disabled` 
`enabled` is based on the `registrations_enabled` checkbox. If this checkbox, is not checked, admin can't not modifiy the existing field`available_slots` and now, we are adding the field `reserved_slots` with the same functionality.

Is this alright?

Reply:
LOL I didn't see it's a variable declared two lines above 🤦‍♂️ sorry 😅